spec/fixtures/test_config_mixin.rb

```diff
@@ -0,0 +1,6 @@
+require 'shanty/config_mixin'
```

Comment:
I just realised that in my haste to raise the PR on Thursday I forgot to remove the other two fixtures - they're actually unnecessary anymore. I was aiming to remove the entire `fixtures` dir because it started this weird trend of the details of a test leaking out across too many files to make sense of at a glance anymore.

See if you can construct this in the `subject` pragma instead:

```ruby
subject { test_config_class.new }
let(:test_config_class) do
  Class.new do
    include Shanty::ConfigMixin
  end
end
```